snapcraft/internal/sources/__init__.py

```diff
@@ -65,6 +65,10 @@
     When building, Snapcraft will set the working directory to be this
     subdirectory within the source.
 
+  - source-recurse-submodule: <boolean>
+
+    Snapcraft will recursively fetch submodules from the source tree by default.
```

Comment:
Perhaps we could rephrase this a little bit to tell what the option does, and then tell what the default is. Something like "Control whether submodules will be recursively fetched or not. The default is to recursively fetch submodules from the source tree."